ReClass.NET/Memory/MemoryBuffer.cs

```diff
@@ -366,5 +366,12 @@ public bool HasChanged(int offset, int length)
 
 			return false;
 		}
+
+
+		public UInt64FloatDoubleData ReadFromBuffer(int offset) => new UInt64FloatDoubleData
```

Comment:
```suggestion
		public UInt64FloatDoubleData InterpretData64(int offset) => new UInt64FloatDoubleData
```
Maybe add the 32 bit version too.

Reply:
Done